From 8568106989303dfc2a1cc93cf8dddcbfb35cf22e Mon Sep 17 00:00:00 2001
From: Quoc Ngo <quoc.ngo.wz@rvc.renesas.com>
Date: Mon, 23 Apr 2018 12:27:44 +0700
Subject: [PATCH 505/628] ARM: DTS: r8a7742: move thermal node after dma node

issue "sh_mmcif ee220000.mmc: could not find pctldev for node /dma-controller@e6700000, deferring probe"
appeared on some iwg21m board, the issue was not investigated deeply, but move thermal node after dma
node can prevent the issue

Signed-off-by: Quoc Ngo <quoc.ngo.wz@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a7742.dtsi | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index a099c07..798412b 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -222,14 +222,6 @@
 		power-domains = <&sysc R8A7742_PD_ALWAYS_ON>;
 	};
 
-	thermal@e61f0000 {
-		compatible = "renesas,thermal-r8a7742", "renesas,rcar-thermal";
-		reg = <0 0xe61f0000 0 0x14>, <0 0xe61f0100 0 0x38>;
-		interrupts = <0 69 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&mstp5_clks R8A7742_CLK_THERMAL>;
-		power-domains = <&sysc R8A7742_PD_ALWAYS_ON>;
-	};
-
 	timer {
 		compatible = "arm,armv7-timer";
 		interrupts = <1 13 (GIC_CPU_MASK_SIMPLE(4) |
@@ -401,6 +393,14 @@
 		dma-channels = <13>;
 	};
 
+	thermal@e61f0000 {
+		compatible = "renesas,thermal-r8a7742", "renesas,rcar-thermal";
+		reg = <0 0xe61f0000 0 0x14>, <0 0xe61f0100 0 0x38>;
+		interrupts = <0 69 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp5_clks R8A7742_CLK_THERMAL>;
+		power-domains = <&sysc R8A7742_PD_ALWAYS_ON>;
+	};
+
 	pfc: pfc@e6060000 {
 		compatible = "renesas,pfc-r8a7742";
 		reg = <0 0xe6060000 0 0x250>;
-- 
2.7.4

